### PR TITLE
removed canopennode xml support

### LINKED ISSUE #103: Remove CanOpenNodeXML (inc bridge) & XDD v1

### PR DESCRIPTION
**Pull request by nimrof**
Think i got it all, but please test it out

Closes #103

### CODE REVIEW

File: EDSEditorGUI/Form1.cs [hunk 1]
```diff
@@ -196,11 +188,8 @@ private void openEDSfile(string path,InfoSection.Filetype ft)
             try
             {
                 EDSsharp eds = new EDSsharp();
-                Device dev; 
 
                 eds.Loadfile(path);
-                Bridge bridge = new Bridge(); //tell me again why bridge is not static?
-                dev = bridge.convert(eds);
 
                 DeviceView device = new DeviceView(eds, network);
```
Comment (nimrof) [inline EDSEditorGUI/Form1.cs:194]: No idea why bridge is used here 🤔

File: README.md [hunk 2]
```diff
@@ -69,10 +69,6 @@ Library
 * Read EDS/DCF/XDC file and parse contents to approprate classes
 * Dump EDS/DCF classes via ToString()
 * Save EDS/DCF classes back to EDS file
-* Read CanOpenNode xml project file
-* Write CanOpenNode xmlproject file
-* Switch formats between EDS/DCF and CanOpenNode XML (note to EDS will result in
```
Comment (nimrof) [inline README.md:74]: Should this line be rewritten or is it okay to remove it?

**Review by CANopenNode — APPROVED** [review]
It seems OK and works as expected at this point.

**Review by trojanobelix — APPROVED** [review]

**Comment by nimrof** [comment]
Thanks for the checks.
Will try to make a pr. to remove xdd v1 before i merge this one

**Comment by trojanobelix** [comment]
@CANopenNode @nimrof:
Without XML Support the default profiles DSP302-NMTMaster.xml and DS301_profile_old.xml can`t be used anymore.
They should be converted to a xpd-file

**Pull request merged**